<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecoveryDialog</class>
 <widget class="QDialog" name="RecoveryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Operation Interrupted</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Some files could not be processed</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="summaryFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="summaryLayout">
      <item>
       <widget class="QLabel" name="successIcon">
        <property name="text">
         <string>✓</string>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string>color: green;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="successCountLabel">
        <property name="text">
         <string>0 files backed up successfully</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="failedIcon">
        <property name="text">
         <string>✗</string>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string>color: red;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="failedCountLabel">
        <property name="text">
         <string>0 files failed</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="failedLabel">
     <property name="text">
      <string>Failed items:</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="failedItemsTree">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>File</string>
      </property>
      <property name="width">
       <number>250</number>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Error</string>
      </property>
      <property name="width">
       <number>300</number>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Retry?</string>
      </property>
      <property name="width">
       <number>60</number>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="retryInfoLabel">
     <property name="text">
      <string>Items marked "Yes" in the Retry column may succeed if you retry.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>color: gray; font-style: italic;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="retryFailedBtn">
       <property name="text">
        <string>Retry Failed</string>
       </property>
       <property name="toolTip">
        <string>Retry all items that might succeed on retry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="continueBtn">
       <property name="text">
        <string>Skip &amp; Continue</string>
       </property>
       <property name="toolTip">
        <string>Skip failed items and continue with the operation</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="abortBtn">
       <property name="text">
        <string>Abort</string>
       </property>
       <property name="toolTip">
        <string>Stop the operation and discard any remaining items</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
